### DISCUSSION TITLE
Stockstatus on Label

### DISCUSSION THREAD

**Discussion by Pixeltrace-GER** [inventree/InvenTree#3739] [Q&A] [answered]
Hi Guys,
I'm so sorry, but I have another question.
I have generated some Labels for my StockItems. One part of the Label should be the StockItem status. 
I have managed to get the status code on the Label, but not the status as text. 

Can you point me to the right direction?

Many thanks, 
Tim

**Comment by SchrodingersGat** [comment]
@Pixeltrace-GER good question - you might be able to use:

```
{{ item.status_text }}
```

Please let me know if that works!

**Reply by Pixeltrace-GER** [reply]
@SchrodingersGat, unfortunately no.
If i change `{{ item.status }}` to `{{ item.status_text }}` I will get no output.

**Comment by SchrodingersGat** [comment, marked as answer]
@Pixeltrace-GER see https://github.com/inventree/InvenTree/pull/3752

I have added the `status_text` function to these models, so (after this PR is merged) you can do the following in your label template:

```html
{% block content %}

<div class='text'>
    Status: {{ item.status }}<br>
    Text: {{ item.status_text }}
</div>

{% endblock %}
```

**Reply by Pixeltrace-GER** [reply]
wow, wonderful. You guys are great. Thank you @SchrodingersGat 
so, with the next stable release, I can add this to the labels.

Thanks
